samples/complex/RetrievalBot/dotnet/RetrievalBot/README.md

```diff
@@ -0,0 +1,155 @@
+﻿# RetrievalBot Sample with Semantic Kernel
+
+This is a sample of a simple Retrieval Agent that is hosted on an Asp.net core web service.  This Agent is configured to accept a request asking for information about Build sessions by Contoso and respond to the caller with an Adaptive Card.
+
+This Agent Sample is intended to introduce you to the Copilot Retrieval API Grounding capabilities. It uses Semantic Kernel with the Microsoft 365 Agents SDK. It can also be used as a the base for a custom Agent that you choose to develop.
+
+***Note:*** This sample requires JSON output from the model which works best from newer versions of the model such as gpt-4o-mini.
+
+## Prerequisites
+
+- [.Net](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) version 8.0
+- [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows)
+- [Bot Framework Emulator](https://github.com/Microsoft/BotFramework-Emulator/releases) for Testing Web Chat.
+- Download and install Visual Studio (I have 2022 version).
+- You need Azure subscription to create Azure Bot Service. If you are MSFT, you get it free. Follow the steps here – Link TBD
+- Download and install Github [Git - Installing Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
```

Comment:
GitHub is a service, not something you install. You can install git, but these days you can assume folks have it already